freezer/freezer_url.go

```diff
@@ -0,0 +1,95 @@
+package freezer
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/uw-labs/freezer"
+	"github.com/uw-labs/straw"
+	"github.com/uw-labs/substrate"
+	"github.com/uw-labs/substrate/suburl"
+)
+
+func init() {
+	suburl.RegisterSink("freezer+dir", newKafkaSink)
+	suburl.RegisterSource("freezer+dir", newKafkaSource)
+	suburl.RegisterSource("freezer+s3", newKafkaSource)
+}
+
+func newKafkaSink(u *url.URL) (substrate.AsyncMessageSink, error) {
```

Comment:
left over from copy pasting stuff I suppose, same for the source below
```suggestion
func newFreezerSink(u *url.URL) (substrate.AsyncMessageSink, error) {
```

Reply:
fixed